src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends FiguresState implements FiguresArea, DrawProperties {
+    private final double radius = FigureSupplier.getRandomProperties();
+
+    public String getName() {
+        String name = "Circle";
+        return "Figure: " + name;
```

Comment:
Method `getFieldName()` must return this field directly, without concatenation with its name, same in all you're getter methods
```suggestion
        return name;
```

Reply:
done